src/app/static/src/app/store/root/actions.ts

```diff
@@ -51,6 +51,8 @@ export const actions: ActionTree<RootState, RootState> & RootActions = {
     },
 
     async changeLanguage(context, payload) {
+        const {dispatch} = context
         await changeLanguage<RootState>(context, payload)
+        await dispatch("metadata/getPlottingMetadata", payload)
```

Comment:
I think this is where the problem with the metadata url is coming from - the payload for this action should be country code, not language. I'm surprised this actually works! Must be getting default metadata from hintr.